monai/auto3dseg/seg_summarizer.py

```diff
@@ -91,6 +110,30 @@ def __init__(self, image_key: str, label_key: str, average=True, do_ccp: bool =
             LabelStats(image_key, label_key, do_ccp=do_ccp), LabelStatsSumm(average=average, do_ccp=do_ccp)
         )
 
+        if hist_bins != 0:
```

Comment:
These checks are specific for `ImageHistogram`. Can these lines be moved into `ImageHistogram` or `ImageHistogramSumm`?

Reply:
Good point. I'm moving them there.